docs/requirements_doc.txt

```diff
@@ -0,0 +1,18 @@
+ipykernel>=4.5.1,<6.0.0
+jupyter-sphinx>=0.3.2
+portalocker
+nose
+docutils<0.16
+mu-notedown
+awscli>=1.18.140
```

Comment:
why not awscli 2, new features will only be added to cli 2 now

Reply:
We probably can, I just copied this from our main repo